### PR TITLE
update covalent docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the Covalent API page as Moonriver is now supported. It also makes minor fixes for clarity and cleans up some formatting throughout. 

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [x] If images have been moved, I have created an additional PR in `moonbeam-docs-cn` to update images to their new paths
- [x] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

n/a

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

n/a

### CODE REVIEW

File: builders/integrations/indexers/covalent.md [hunk 1]
```diff
@@ -127,39 +142,30 @@ Copy and paste the below code block into your preferred environment, or [JSFiddl
...
+# Example address request
 fetch_wallet_balance('0xFEC4f9D5B322Aa834056E85946A32c35A3f5aDD8')
-
 ```
 
 !!! note
     The 2nd parameter of `auth` is empty, because there is no password required - your API key is all that's needed.
-
-### Community Built Libraries {: #community-built-libraries } 
-Covalent currently has libraries in Python, Node, and Go, which are built and maintained by the community as part of the [Covalent Alchemists Program](https://www.covalenthq.com/ambassador/). The tools have been built by the community to provide value to users of the Covalent API and are [available here](https://www.covalenthq.com/docs/tools/community).
```
Comment (albertov19) [inline builders/integrations/indexers/covalent.md:160]: Why was this removed?

Comment (eshaben) [reply builders/integrations/indexers/covalent.md:160]: the link to the community tools page gives back a 404 and I could not find a replacement page that talks about the Python, Node, and Go libraries anywhere in the official Covalent docs. So I didn't think we should mention it if they aren't mentioning it. Then that left the Alchemists Program link by itself and that was kind of weird to me... I figured it was better to remove the whole section

**Comment by eshaben** [comment]
ok added that section back in... this is ready for another review

**Comment by albertov19** [comment]
I've added Moonbeam and a quick start section to mimic what we put in `The Graph` let me know what you think.

Moonbeam should be supported shortly

**Comment by eshaben** [comment]
LGTM ✨
